helm_deploy/prisoner-content-hub-frontend/templates/ingress.yaml

```diff
@@ -14,11 +14,11 @@ metadata:
 spec:
   tls:
   {{- range .Values.ingress.hosts }}
+  {{- $certSecret := .cert_secret -}}
```

Comment:
I understand more formatting now and I always wanted these here 😁

Reply:
Ahhh yeah, I've been bitten by that a couple of times 🤦. Elegant helm indentation != valid YAML ☹️.